Resources/Private/Config/resource.php

```diff
@@ -2,19 +2,21 @@
 
 $publicPath = \TYPO3\CMS\Core\Core\Environment::getPublicPath();
 
+$defaultConnection = $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default'];
+$fallbackConnection = $GLOBALS['TYPO3_CONF_VARS']['DB'];
```

Comment:
As Aimeos only supports TYPO3 10 and 11, you can remove the fallback now. Then, you don't need to stack the `??` operator